src/qml/pages/node/NodeRunner.qml

```diff
@@ -0,0 +1,43 @@
+// Copyright (c) 2022 The Bitcoin Core developers
+// Distributed under the MIT software license, see the accompanying
+// file COPYING or http://www.opensource.org/licenses/mit-license.php.
+
+import QtQuick 2.15
+import QtQuick.Controls 2.15
+import QtQuick.Layouts 1.15
+import "../../controls"
+import "../../components"
+
+Page {
+    background: null
+    clip: true
+    property alias navRightDetail: navbar.rightDetail
+    header: NavigationBar {
+        id: navbar
+    }
+    ColumnLayout {
+        spacing: 0
+        anchors.fill: parent
+        ColumnLayout {
```

Comment:
Here's another double ColumnLayout.  I think we should reduce the layers here or use a different top level Item.

Reply:
maybe we can leave it to a follow-up pr to address the nested columnlayouts throughout the code?